Doc: Add docs for displaying links in the dashboard-controls menu (#115201)
grafana/grafana: docs/sources/as-code/observability-as-code/schema-v2/links-schema.md

| targetBlank | bool. If true, the link will be opened in a new tab. Default is `false`. |
| includeVars | bool. If true, includes current template variables values in the link as query params. Default is `false`. |
| keepTime    | bool. If true, includes current time range in the link as query params. Default is `false`. |

> | placement?  | string. Placement can be used to display the link somewhere else on the dashboard other than above the visualizations. Accepted value: `inControlsMenu` - renders the link in the dashboard controls dropdown menu. |

Reviewer: ```suggestion
| placement?  | string. Use placement to display the link somewhere else on the dashboard other than above the visualizations. Use  the `inControlsMenu` parameter to render the link in the dashboard controls dropdown menu. |
```
